libs/daemon/client/src/mill/client/ServerLauncher.java

```diff
@@ -323,12 +346,15 @@ private static ServerLaunchOutputs readOutputs(Path daemonDir) throws IOExceptio
    * @return a PumperThread that processes the output/error streams from the server
    */
   static PumperThread startStreamPumpers(
-      InputStream socketInputStream, OutputStream socketOutputStream, Streams streams) {
+      InputStream socketInputStream,
+      OutputStream socketOutputStream,
+      Streams streams,
+      String name) {
```

Comment:
I think we don't need the `name` anymore, since there's only one callsite of this method in the entire codebase

Reply:
Right now, yes, but having the name allows you to quickly identify what for these pumpers are without having to look at codebase. Is there a downside to having a name here?

Reply:
If we're not using it, we should remove it, and add it back later as necessary. In general, we try to avoid code that might be useful _later_ hanging around in the codebase, which has the nice benefit of meaning all the code in the codebase is useful _now_ which makes it easier to understand since you know any piece of code _must_ be used